docs/paths/request-bodies.md

```diff
@@ -1,5 +1,42 @@
 # Request Bodies
 
+Generate a request body with this command:
+
 ```bash
 php artisan openapi:make-requestbody StoreUser
 ```
+
+You can refer to a schema by implementing `use Vyuldashev\LaravelOpenApi\Contracts\Reusable` on the schema and adding it to the request body like so:
+
+```php
+class UserCreateRequestBody extends RequestBodyFactory
+{
+    public function build(): RequestBody
+    {
+        $userSchema = new UserSchema();
+
+        return RequestBody::create('UserCreate')
+            ->description('User data')
+            ->content(
+                MediaType::json()->schema($userSchema->ref())
```

Comment:
`UserSchema::ref()` instead of `$userSchema->ref()`

Reply:
Oh, of course…I didn’t look at the `Referencable` trait to notice that.